neonvm/controllers/vm_controller_test.go

```diff
@@ -111,6 +111,7 @@ var _ = Describe("VirtualMachine controller", func() {
 					MaxConcurrentReconciles: 1,
 					QEMUDiskCacheSettings:   "cache=none",
 					DefaultMemoryProvider:   vmv1.MemoryProviderDIMMSlots,
+					MemhpAutoMovableRatio:   "301",
```

Comment:
Should have 801 everywhere?

Reply:
My thinking here was that 301 is a more "safe" default, but we know we can get away with 801 in practice (kind of like the cache=none situation). (so, this 301 here matches the default of 301 for the neonvm-controller flag, which we then always override)

Thoughts?

Reply:
It's just that this PR request has both 301 and 801 in different places. Just wondering is that is intentional, ok with me either way.